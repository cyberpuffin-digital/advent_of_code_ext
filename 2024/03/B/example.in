xmul(2,4)&mul[3,7]!^don't()_mul(5,5)+mul(32,64](mul(11,8)undo()?mul(8,5))don't()ljew
xmul(2,4)&mul[3,7]!^do()_mul(5,5)+mul(32,64](mul(11,8)undon't()?mul(8,5))!$_&^!)do()-mul(8,64)+08!%.>_mul(12,82)