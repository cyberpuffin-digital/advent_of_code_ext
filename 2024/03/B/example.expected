This time, the sum of the results is 48 (2*4 + 8*5).
48 + 48 + (8*64 + 12*82) == 1592

---

xmul(2,4)&mul[3,7]!^don't()_mul(5,5)+mul(32,64](mul(11,8)undo()?mul(8,5))don't()ljew

mul(2,4) - don't() - mul(5,5) - mul(11,8) - do() - mul(8,5) - don't()
2*4 + 8*5 = 48 - disabled

---

xmul(2,4)&mul[3,7]!^do()_mul(5,5)+mul(32,64](mul(11,8)undon't()?mul(8,5))!$_&^!)do()-mul(8,64)+08!%.>_mul(12,82)

mul(2,4) - do() - mul(5,5) - mul(11,8) - don't() - mul(8,5) - do() - mul(8,64) - mul(12,82)

48 + 5*5 + 11*8 + 8*64 + 12*82 = 1657 - enabled
